README.md

```diff
@@ -95,6 +95,7 @@ Support for Erlang tools, including rebar3, EUnit and Dialyzer
 - `erlang.includePaths` - Include paths are read from rebar.config, and also standard set of paths is used. This setting is for special cases when the default behaviour is not enough
 - `erlang.linting` - Enable/disable dynamic validation of opened Erlang source files
 - `erlang.codeLensEnabled` - Enable/Disable CodeLens
+- `erlang.compressLargeEtsTables` - Enable/disable compression of large ETS lookup tables (mainly used for code navigation) to consume less memory. Please note, it might impact performance.
```

Comment:
compressLargeEtsTables is not easy to understand for "users". and "please note it might impact performance" can affraid users

proposition : 'erlang.cacheManagement' = default, compressed, file 
description : Define caching usage for code navigation, parsing result,...depending on your project size

is it make sense ?